org.nifty-site-manager.nsm.json

```diff
@@ -0,0 +1,24 @@
+{
+    "app-id": "org.flatpak.nsm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "nsm",
+    "modules": [
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
+            "build-commands": [
+				"make",
+				"make install"
+            ],
+            "sources": [
+                {
+					"type": "archive",
+					"url": "https://github.com/nifty-site-manager/nsm-flatpak/raw/master/nsm.tar.xz",
```

Comment:
This is the incorrect source, you should only have a single repo: https://github.com/nifty-site-manager/nsm

Reply:
I would have to put files for flathub/snapcraft/chocolatey/homebrew if it was all in the same repo which is a little confusing for people. Seemed nicer for me to just have a separate repo for each platform?

Reply:
Nothing is unique to Flatpak. It should be building your normal sources.